doc/customization.md

```diff
@@ -57,6 +57,18 @@ If you use android, you need to change debug server by:
 
 * Note that some aspects of React Native hard-code the port to the default as specified in [this issue](https://github.com/facebook/react-native/issues/9145).
 
+## Setting up React Native global CLI
+
+To use React Native CLI installed globally:
```

Comment:
I think we should add more details here. It should be clear what this property do. I think more detailed description is needed here, and maybe at least one usage example.